### PR TITLE
Added new topic

### PR DESCRIPTION
**Pull request by michaelbukachi**
Added a new topic about **Maintaining state while running pytests**

### CODE REVIEW

File: content/pytest_maintaining_state.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+---
+
+## Pre-requisites
+ > - Beginner level Python
+ > - Python2* or Python3* installed
+ > - Basic knowledge on TDD and python UnitTesting
+ > - Pytest installed
+ 
+ 
+ #Intro
```
Comment (mrmoje) [inline content/pytest_maintaining_state.md]: Nuke the space at col 1 (for this line L20 to L48) to fix formatting for the headings and codeblocks in between.
image

**Review by mrmoje — CHANGES_REQUESTED** [review]
Good stuff! Just make the minor change recommended and we are good for merge

File: content/pytest_maintaining_state.md (hunk 1 above)
Comment (michaelbukachi) [reply content/pytest_maintaining_state.md]: Okay. I'll make the change and do another pull request

**Comment by michaelbukachi** [comment]
@mrmoje  I've made the necessary changes

**Review by mrmoje — APPROVED** [review]

**Pull request merged**